rotkehlchen/db/updates.py

```diff
@@ -178,6 +228,10 @@ def check_for_updates(self) -> None:
         if local_rpc_nodes_version < latest_rpc_nodes_version:
             self.update_rpc_nodes(latest_rpc_nodes_version)
 
+        latest_contracts_version = remote_information[UpdateType.CONTRACTS.value]['latest']
+        if local_contracts_version < latest_contracts_version:
+            self.update_contracts(latest_contracts_version)
```

Comment:
This is repeating over the other differnt  types. Why not move this to a function that iterates UpdateType and with an if clause you choose what function to execute?